xonsh/environ.py

```diff
@@ -539,6 +539,13 @@ def _replace_home(x):
 
 _replace_home_cwd = lambda: _replace_home(builtins.__xonsh_env__['PWD'])
 
+def _collapsed_pwd():
+    pwd = _replace_home_cwd().split('/')
```

Comment:
I think that `'/'` should be replaced with `os.sep` everywhere in this function.